source/sessions/driver-sessions.rst

```diff
@@ -137,7 +137,7 @@ Applications start a new session like this:
 
 .. code:: typescript
 
-    options = new SessionOptions(...);
+    options = new SessionOptions(/* various settings */);
```

Comment:
These style changes make this PR harder to review. It would have been better to put split them into a separate PR.

Reply:
Very good point! fixed.